src/gcp_scanner/crawl.py

```diff
@@ -695,9 +691,9 @@ def get_spanner_instances(project_id: str,
         parent=f"projects/{project_id}")
     while request is not None:
       response = request.execute()
-      for instance in response.get("instances", []):
-        spanner_instances_list.append(instance)
-
+      spanner_instances_list = [instance 
+                                for instance in response.get("instances", [])]
+                                
```

Comment:
Trim trailing whitespace

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.